web/src/pages/Home.tsx

```diff
@@ -68,7 +35,9 @@ function Home() {
 
   return (
     <section className="page-wrapper home">
-      <div className="banner-wrapper">{state.shouldShowUpdateVersionBanner && <UpdateVersionBanner />}</div>
+      <div className="banner-wrapper">
+        <UpdateVersionBanner />
```

Comment:
I think banners should control themselves to show or not. Otherwise, the `Home` component will be full of banner-related logic.